tests/test_sentences_endpoints.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+import json
+from .factory import insertDummySentences, removeCollections
+from app import create_app
+from app.models import *
+import sys
+sys.path.append("..")
+
+
+@pytest.yield_fixture
+def app():
+    app = create_app("testing")
+    yield app
+
+
+@pytest.fixture
+def test_cli(loop, app, sanic_client):
+    return loop.run_until_complete(sanic_client(app))
+
+
+@pytest.fixture
+def generateData(request):
+    # adding dummy sentences
+    insertDummySentences()
+
+    def fin():
+        removeCollections()
+
+    request.addfinalizer(fin)
+
+
+async def test_getSentences_with_noParam_shouldPass(test_cli, generateData):
+    resp = await test_cli.get('/sentences')
+    resp_json = await resp.json()
+    assert resp_json["total_results"] == 10
+    assert len(resp_json["results"]) == 10
```

Comment:
how was 10 chosen? what does it represent?

Reply:
All my dummy contents come from files in `db/dummy/`,  I actually have 10 objects for the sentences that I inserted.